ReactAndroid/src/main/java/com/facebook/react/animated/InterpolationAnimatedNode.java

```diff
@@ -12,6 +12,32 @@
  */
 /*package*/ class InterpolationAnimatedNode extends ValueAnimatedNode {
 
+  /* package */ enum ExtrapolateType {
+    IDENTITY("identity"),
+    CLAMP("clamp"),
+    EXTEND("extend");
+
+    private final String mName;
+
+    ExtrapolateType(String name) {
+      mName = name;
+    }
+
+    public static ExtrapolateType fromString(String name) {
+      for (ExtrapolateType type : ExtrapolateType.values()) {
```

Comment:
My personal preference would be not to use enums for such a simple case. Enums on android are substantially slower than on regular JVM, also here you allocate an extra and unnecessary iterator which is a pattern we try to avoid in oder places. So I'd prefer to at least change this piece